specification/vmware/resource-manager/readme.csharp.md

```diff
@@ -5,11 +5,11 @@ Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azur
 
 ```yaml $(csharp)
 csharp:
-  package-version: 0.3.0
+  package-version: 0.1.0
```

Comment:
Just FYI that the final package version is determined by the `<Version>` tag in your C# project's `*.csproj` file, so updating here is not necessary.

Reply:
Good to know. Thanks.